src/main/MainScreenWithTabs.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+// @ts-check
```

Comment:
What's this for? It looks like it's for turning on some kind of TypeScript-related checking.

We use Flow types in our codebase -- I definitely recommend setting up your dev environment to give you feedback on Flow type errors, and also errors from ESLint (which we also use). Let's discuss in chat; I'll @-mention you in a thread. (We don't quite yet have good docs on how to set this up.)

Once you have that working, if there are additional useful checks that this would provide that we don't get from Flow, or from ESLint (which we also have), I'd be quite interested in trying that out! Let's not mix it into unrelated commits like this one, though. And if we do do it, we'll want to do it systematically on large swaths of the code. :-)

Reply:
Sorry just did that for my convenience I will remove that.

Reply:
Yes -- let's not mix it into unrelated commits like this one.